views/home.hbs

```diff
@@ -0,0 +1,9 @@
+<form action="/artist-search" , method="GET">
+  <input
+    type="search"
+    id="search"
+    name="q"
```

Comment:
the `name` defines what the query parameter name will be.
if you're looking for a `artistName` then the name should be `artistName`